python/py-coveralls/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-coveralls
```

Comment:
There are two similarly named but (as far as I can tell) distinct projects:

* http://github.com/z4r/python-coveralls
* http://github.com/coveralls-clients/coveralls-python

I suggest using the name py-python-coveralls (as inelegant as that is) in order to avoid any conflicts in the future, if someone else packages coveralls-python

Reply:
Groan. Thanks for pointing this out—I’ll make that change.

Reply:
Done thanks.